src/static/config.json

```diff
@@ -6,7 +6,7 @@
         "workspace": "/ws",
         "search": "/searchapi2/rpc",
         "service_wizard": "/service_wizard",
-        "user_profile": "/user_profile",
+        "user_profile": "/user_profile/rpc",
```

Comment:
This seems new. Is that a new endpoint for the user_profile service?

Reply:
I am not sure, this line was already in the config, but it was not in use, and to work with the `KBaseServiceClient` from `@kbase/narrative-utils` it needs the `/rpc` as far as I am aware. The dashboard was using the `DynamicServiceClient` with the `bff` service previously, but @eapearson suggested using the `user_profile` service directly, and that seemed simpler overall which led to this change.

Reply:
Yeah... I think user_profile is one of those odd services that the endpoint is an extra level deeper.

Reply:
Actually, endpoints shouldn't be embedded in the codebase at all, other than perhaps testing configuration. 
This should come from a config stored in kbase gitlab and provided to the container by the deployment mechanism.

Reply:
Erik's suggestion is definitely more in line with more in line with modern practice, but I am not sure that any environment variables are configured to be passed to the front end at present.